paasta_tools/autoscaling_lib.py

```diff
@@ -774,6 +775,9 @@ def scale_aws_spot_fleet_request(resource, current_capacity, target_capacity, so
             log.info("Decreasing spot fleet capacity from {0} to: {1}".format(current_capacity, new_capacity))
             if not set_spot_fleet_request_capacity(sfr_id, new_capacity, dry_run):
                 log.error("Couldn't update spot fleet, stopping autoscaler")
+                if not dry_run:
+                    log.info("Undraining {0}".format(slave_to_kill['pid']))
```

Comment:
Wouldn't it be more useful to move the log statements outside of the if statement so you can see in dry_run mode that PaaSTA would normally undrain the host at this point?